src/application/event_handler.rs

```diff
@@ -44,4 +49,17 @@ impl EventHandler {
             _ => (),
         };
     }
+
+    fn handle_prompt_key_press(key: KeyEvent, app: &mut App) {
+        match key.code {
+            KeyCode::Enter => {
+                app.confirm_previous_action();
+                app.input_mode = InputMode::Command;
+            }
```

Comment:
You should probably change app.previous_action at this point, since it's no longer ClearAllTasks

Reply:
done